frontend/src/utils/showToastOnError.ts

```diff
@@ -0,0 +1,31 @@
+import type { ApiError } from "../client"
+
+interface ErrorDetail {
+  msg: string
+}
+
+interface ApiErrorBody {
+  detail?: string | ErrorDetail[]
+}
+
+// These variables are not used, they are just type definition
```

Comment:
We should aim for not adding eslint-disable comments. What exactly is the issue here?

Reply:
I believe it was:
`title`, `description`, `status` was declared but never used.

Reply:
Oh, thats annoying. One solutino would be to have the types in a types.ts file

Reply:
I think declaring the type from `types.ts` might still throw codacy warning even if we are exporting it. The warning was not the type variable itself not being used, but rather the parameter variables.

There's no `types.ts` file yet that we can modify - there's `types.ts` from the `client` directory but this only stores API-related types and auto-generated. We could definitely address in future PR to introduce another `types.ts` in our codebase.